src/components/common/CryptoExchangeWalletListRow.js

```diff
@@ -174,8 +174,12 @@ class CryptoExchangeWalletListRow extends Component<Props, LocalState> {
   }
   renderWallet = () => {
     const { wallet, searchFilter, isMostRecentWallet, currencyCodeFilter, allowedCurrencyCodes, excludeCurrencyCodes } = this.props
-    const checkAllowedCurrencyCodes = allowedCurrencyCodes ? allowedCurrencyCodes.find(currencyCode => currencyCode === wallet.currencyCode) : true
-    const checkExcludeCurrencyCodes = excludeCurrencyCodes ? excludeCurrencyCodes.find(currencyCode => currencyCode === wallet.currencyCode) : false
+    const checkAllowedCurrencyCodes = allowedCurrencyCodes
+      ? allowedCurrencyCodes.find(currencyCode => currencyCode.toUpperCase() === wallet.currencyCode.toUpperCase())
+      : true
+    const checkExcludeCurrencyCodes = excludeCurrencyCodes
+      ? excludeCurrencyCodes.find(currencyCode => currencyCode.toUpperCase() === wallet.currencyCode.toUpperCase())
+      : false
     const searchFilterLowerCase = searchFilter.toLowerCase()
     const nameString = wallet.name.toLowerCase()
     const currencyNameString = wallet.currencyNames[wallet.currencyCode].toLowerCase()
```

Comment:
Convert currency arrays on walletListModal to uppercase

Reply:
fixed

Reply:
This does not appear to be fixed. Currency codes should be preprocessed to uppercase before being used in walletListModal or passed to walletListRow